### DISCUSSION TITLE
error: RPC failed; curl 55 SSL_write() returned SYSCALL, errno = 10053

### DISCUSSION THREAD

**Discussion by sahimsakir** [community/community#37764] [Other Feature Feedback, Questions, & Ideas]
git push --set-upstream origin main
fatal: HttpRequestException encountered.
   An error occurred while sending the request.
Username for 'https://github.com': sahimsakir
Counting objects: 14710, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (14172/14172), done.
error: RPC failed; curl 55 SSL_write() returned SYSCALL, errno = 10053
fatal: The remoWriting objectte s:end hung  up un  1% (288e/14710),x 984.pe00ct Kedly
Writing objects: 100% (14710/14710), 2.21 GiB | 25.54 MiB/s, done.
Total 14710 (delta 1499), reused 0 (delta 0)
fatal: The remote end hung up unexpectedly
Everything up-to-date

why this error is showing and how to solve it?

**Comment by NadimMahmudbd7** [comment]
git config --global http.postBuffer 1048576000
